shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -51,6 +51,17 @@
           <button class="close-button" title="Close {{title}}"></button>
         </li>
         {{/each}}
+
+        {{#with accountExpires}}
+          <li><div class="demo-notice">
+            <p>This demo expires in: <span class="countdown {{#if urgent}}urgent{{/if}}">{{countdown}}</span></p>
+
+            <p>To make your data permanent, create an account:
+              <button class="sign-in">Create account »</button></p>
+
+            <p class="self-host"><a href="https://sandstorm.io/install" target="_blank">Or, get Sandstorm on your own server (it's open source) »</a></p>
```

Comment:
Add a trailing `/` to avoid a redirect (not that it matters much) to https://sandstorm.io/install/